src/auto-evo/run_parameters.cpp

```diff
@@ -105,8 +105,32 @@ std::string
     RunParameters::makeSummaryOfExternalEffects() const
 {
     std::stringstream sstream;
+    std::vector<std::tuple<Species::pointer, int, std::string>>
+        combinedExternalEffects;
 
     for(const auto& [species, amount, eventType] : m_externalEffects) {
+        int combinedIdx = 0;
+        bool combined = false;
+
+        for(const auto& [combinedSpecies, combinedAmount, combinedEventType] :
+            combinedExternalEffects) {
+            if(species == combinedSpecies && eventType == combinedEventType) {
+                combinedExternalEffects.at(combinedIdx) =
+                    std::make_tuple(combinedSpecies, combinedAmount + amount,
```

Comment:
I think this could be changed to directly modify the amount
by removing the const from line 115, then also combinedIdx wouldn't be needed.